README.md

```diff
@@ -61,6 +61,7 @@ The `schema`, `tables`, and `bucket` flags also are critical. `granularity` is r
 - `force`: refresh the data even if the data date is after the current `s3` input date
 - `date`:  the date string for the data in question
 - `config`: override of the usual auto-discovery of the config
+- `delimiter`: required to use CSV files, what the file is delimited in (likely use the '|' pipe character as that is AWS' default)
```

Comment:
hmm, I think, if we can get away with it, we should go for the pipe since that is Amazon's default and a bit easier to work with as humans generally. I'm hoping that the issue we saw really was a pathological case...

But I haven't thought too much about it, let's talk it over!